Tests/SystemInfo/Collector/ComposerSystemInfoCollectorTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/**
+ * File containing the ComposerSystemInfoCollectorTest class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzSupportToolsBundle\Tests\SystemInfo\Collector;
+
+use EzSystems\EzSupportToolsBundle\SystemInfo\Collector\ComposerSystemInfoCollector;
+use EzSystems\EzSupportToolsBundle\SystemInfo\Value\ComposerSystemInfo;
+use PHPUnit_Framework_TestCase;
+
+class ComposerSystemInfoCollectorTest extends PHPUnit_Framework_TestCase
+{
+    /**
+     * @var ComposerSystemInfoCollector
+     */
+    private $composerCollector;
+
+    public function setUp()
+    {
+        $this->composerCollector = new ComposerSystemInfoCollector(__DIR__ . '/_fixtures/');
```

Comment:
Nice to know, but also not sure if better. Should maybe have a unified scheme for all/most tests, and `_fixtures` seems most used.